src/app/shared/search/filters/search-filters.component.ts

```diff
@@ -33,7 +33,8 @@ export class SearchFiltersComponent implements OnInit {
     caseType?: CaseType,
     formGroup?: FormGroup,
     caseState?: CaseState,
-    page?: number
```

Comment:
sooner or later we have to introduce a proper type here with properties and utility methods

Reply:
this code fragment is from another ticket and got merged here accidentally